packages/story-editor/src/app/canvas/canvasProvider.js

```diff
@@ -47,6 +47,8 @@ function CanvasProvider({ children }) {
   const [eyedropperPixelData, setEyedropperPixelData] = useState(null);
   const [isEyedropperActive, setIsEyedropperActive] = useState(null);
   const [eyedropperCallback, setEyedropperCallback] = useState(null);
+  const [pageCanvasData, setPageCanvasData] = useState(null);
+  const [pageCanvasPromise, setPageCanvasPromise] = useState(null);
```

Comment:
Remember to mock the in the unit tests - there are a few failures due to undefined functions.